README.md

```diff
@@ -336,27 +102,12 @@ The `.subscribe` method can take [a few different forms](https://github.com/Reac
    npm run lint:fix
    ```
 
-### Developing with the Example App
```

Comment:
Might be nice to add a reference to `DevSandbox` here.

Reply:
Good idea, I'll add that.